fecgov/openFEC
# remove duplicated rows of staging.ref_rpt_tp in sample_db.sql

## Summary (required)

- Associated with  [_#3400_](https://github.com/fecgov/openFEC/issues/3400)

- There should be 49 rows in ref_rpt_tp table, but smple_db.sql has 102 rows. This PR wil remove the duplicated rows

## How to test the changes locally

- Pull the branch
- import sample_db to local database cfdm_test
- check table ref_rpt_tp
